meson.build

```diff
@@ -0,0 +1,26 @@
+project(
+  'starfit',
+  'fortran',
+  'c',
+# Run command to rewrite version number:
+# meson rewrite kwargs set project / version $(git describe --tags)
+version : '0.0.0',
+default_options : ['buildtype=debugoptimized', 'optimization=3']
+)
+
+py_mod = import('python')
+py3 = py_mod.find_installation('python3')
```

Comment:
I would pass `pure: false` to find_installation. This is available since Meson 0.64.0 and ensures that pure python sources are installed to the same location as the compiled extensions (it's generally, though not always, bad to have both purelib and platlib at the same time).

This overrides the default behavior of py3.install_sources and py3.get_install_dir (which have their own `pure: ` kwargs as well).